pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/io/stream/ReadableStreamingDataTest.java

```diff
@@ -232,4 +271,42 @@ void readFileThatDoesntExist() throws IOException {
         Files.delete(file);
         assertThrows(IOException.class, () -> new ReadableStreamingData(file));
     }
+
+    private static class TestReadeableSequentialData implements ReadableSequentialData {
```

Comment:
What's the purposed of this class?

Reply:
The purpose is to test default implementations of
`ReadableSequentialData#readBytes(ByteBuffer)`  and `ReadableSequentialData#readBytes(BufferedData)`

Reply:
Yeah, it seems to delegate all the operations to a given object.

A little javadoc would help understand why it's even needed.

Reply:
Sure, let me add it

Reply:
Done